docs/framework/migration-guide/retargeting/4.0-4.5.1.md

```diff
@@ -9,8 +9,6 @@ author: "chlowell"
 
 [!INCLUDE[versionselector](../../../../includes/migration-guide/retargeting/versionselector.md)]
 
-[!INCLUDE[introduction](../../../../includes/migration-guide/retargeting/introduction.md)]
```

Comment:
I don't know how much people rely on this info on each page to understand the scope info but I think we can publish and see if we get any negative feedback